--- a/src/usr/autoboot.c
+++ b/src/usr/autoboot.c
@@ -71,6 +71,7 @@
 #define NORMAL	"\033[0m"
 #define BOLD	"\033[1m"
 #define CYAN	"\033[36m"
+#define BLUE	"\033[34m"
 
 /** The "scriptlet" setting */
 const struct setting scriptlet_setting __setting ( SETTING_MISC, scriptlet ) = {
@@ -538,11 +539,10 @@
 	 * do so.
 	 *
 	 */
-	printf ( NORMAL "\n\n" PRODUCT_NAME "\n" BOLD PRODUCT_SHORT_NAME " %s"
+	printf ( NORMAL "\n\n" PRODUCT_NAME "\n" BOLD PRODUCT_SHORT_NAME
 		 NORMAL " -- " PRODUCT_TAG_LINE " -- "
-		 CYAN PRODUCT_URI NORMAL "\nFeatures:", product_version );
+		 CYAN PRODUCT_URI NORMAL "\n"
+		 BOLD BLUE "https://bdisk.square-r00t.net/" NORMAL "\n" );
 	printf ( "\n" );
 
 	/* Boot system */
